data-prepper-logstash-configuration/src/main/java/org/opensearch/dataprepper/logstash/mapping/amazon_es.mapping.yml

```diff
@@ -0,0 +1,10 @@
+pluginName: opensearch
+mappedAttributes:
+  hosts: hosts
+  user: username
+  password: password
+  index: index
+additionalAttributes:
+  aws_sigv4: true
+  aws_region: us-east-1
```

Comment:
This is actually the `region` property in the `amazon_es` plugin. So, I think you need to add the following into `mappedAttributes`:
```
aws_region: region
```

Reply:
Shoudn't we take the default value of region here?
Reference: https://github.com/opensearch-project/data-prepper/tree/main/data-prepper-plugins/opensearch

Reply:
We need to map the attribute because users may have it set.

Regarding the defaults, we could use `additionalAttributes` as defaults from Logstash which are applied into. But, the logic would have to be that the `additionalAttributes` are applied first, and then the `mappedAttributes` overwrite those values.